CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - [#7296](https://github.com/apache/trafficcontrol/pull/7296) *Traffic Portal* New configuration option in `traffic_portal_properties.json` at `deliveryServices.exposeInactive` controls exposing APIv5 DS Active State options in the TP UI.
 - [#7332](https://github.com/apache/trafficcontrol/pull/7332) *Traffic Ops* Creates new role needed for TR to watch TO resources.
 - [#7322](https://github.com/apache/trafficcontrol/issues/7322) *t3c Adds support for anycast on http routed edges.
+- [#7364](https://github.com/apache/trafficcontrol/issues/7364) *Traffic Portal* TPv2 Created Coordinates Page Parity.
```

Comment:
We haven't been adding TPv2 (only) stories to the changelog since it's experimental/not released.

Reply:
removed